fcl/dunefd/gen/supernova/prodmarley_nue_es_flat_radiological_decay0_dune10kt_halfActiveVol_1x2x6.fcl

```diff
@@ -3,4 +3,6 @@
 
 physics.producers.marley:      @local::dune_marley_nue_es_flat_halfActiveVol
 
+physics.simulate: [ rns, marley, @sequence::dunefd_hd_backgrounds_1x2x6_v3_6 ] #bg model to emulate lateral APA
```

Comment:
This fcl did not simulate backgrounds before?

Reply:
Same thing as the other comment.